src/federation.rs

```diff
@@ -257,7 +257,7 @@ mod tests {
 
     #[test]
     fn test_signers() {
-        let federation = Federation::new(TEST_KEYS.pubkeys()[4], 0, 3, node_vss(0));
+        let federation = Federation::new(TEST_KEYS.pubkeys()[0], 0, 3, node_vss(0));
 
         let mut pubkeys = TEST_KEYS.pubkeys();
         pubkeys.sort_by(|a, b| {
```

Comment:
You can also remove this line(263-267) because variable 'pubkeys' has already be sorted.

Reply:
fixed at 8f34afb2e669a5d80929f2fe1c21ad6efd6563d5